### ISSUE TITLE
Translation of aria-label strings

### ISSUE THREAD

**Issue by roisag** [liedekef/fdatepicker#5]
This is not a classic bug but suggestion that come to my mind after testing with screen reader. To be more WCAG compliant I think that option to translate this aria-labels strings should be add:

fdatepicker.js 
457: popup.setAttribute('aria-label', 'Date picker');
462:  aria-label="Previous"
464: aria-label="Next"
1794: ariaLabel = `${title}, click to select month`;
1797: ariaLabel = `${title}, click to select year`;

**Comment by liedekef** [comment]
agreed and changed in the code. Released 3.0.50 for this.
